src/Api/FilterLocatorTrait.php

```diff
@@ -62,5 +62,7 @@ private function getFilter(string $filterId)
         if ($this->filterLocator instanceof FilterCollection && $this->filterLocator->offsetExists($filterId)) {
             return $this->filterLocator->offsetGet($filterId);
         }
+
+        return null;
```

Comment:
I thought that php was returning null when nothing is specified?

Reply:
That's correct.

Reply:
IMO we can ditch this 2 `return null`. Apart from that these are really good code improvements.

Reply:
Since PHP 7.1, with the introduction of `void`, this behavior has changed. If you typehint with `: ?object` you **must** explicitly return null. If you typehint with `: void` you **must not** return null.

We don't use nullable nor void return types right now because we still support PHP 7.0, but one day we'll move forward. So I'm to keep these changes.

Reply:
We have to differentiate `null` (`return null`) from `void` (`return;`). In fact, that's what we actually do.

Reply:
See https://github.com/symfony/symfony/issues/17201.

Reply:
I wasn't aware of that thanks. Also TIL isset accepts multiple arguments. \o/ :heart: u